### PR TITLE
Align terminal command ids with VS Code. Fixes #12084

### LINKED ISSUE #12084: Align Terminal Command Ids with VS Code
### Feature Description:
Currently, the command ids of terminal-related commands are not aligned with VS Code: for example, we have `teminal:new` instead of `workbench.action.terminal.new`. 

The list of terminal-related commands in VS Code can be found in the "Keyboard Shortcut" preferences.

### PR DESCRIPTION
**Pull request by tsmaeder**
#### What it does
This PR aligns the command ids of commands related to terminals with their counterparts in VS Code. 
The change includes a "drive-by" fix for the file open dialog that I needed for testing "Open in Terminal".

Fixes #12084

Contributed on behalf of STMicroelectronics

#### How to test
The attached extension contributes commands that invokes each terminal command via its VS Code id.

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by tsmaeder** [comment]
Here's the extension:

[terminalcomms-0.0.1.zip](https://github.com/eclipse-theia/theia/files/10557617/terminalcomms-0.0.1.zip)

and the source:

[terminalcomms.zip](https://github.com/eclipse-theia/theia/files/10557624/terminalcomms.zip)

**Comment by tsmaeder** [comment]
@vince-fugnitto the promised follow-up to https://github.com/eclipse-theia/theia/pull/12066

File: packages/plugin-ext/src/plugin/known-commands.ts [hunk 1]
```diff
@@ -63,6 +65,9 @@ export namespace KnownCommands {
         }
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const ID = (args: any[]) => args;
```
Comment (msujew) [inline packages/plugin-ext/src/plugin/known-commands.ts:69]: Minor: This is supposed to be an identity function, right? I would usually associate `ID` with an `Identifier`, not an `Identity`.
```suggestion
    const identity = (args: any[]) => args;
```

File: packages/plugin-ext/src/plugin/known-commands.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ import {
     isModelCallHierarchyOutgoingCall, toCallHierarchyOutgoingCall, fromTextDocumentShowOptions
 } from './type-converters';
 
+import { URI as TheiaURI } from '@theia/core/lib/common/uri';
```
Comment (vince-fugnitto) [inline packages/plugin-ext/src/plugin/known-commands.ts:30]: minor: I'd remove the extra line and group the imports together.

**Review by vince-fugnitto — APPROVED** [review]
LGTM :+1: I confirmed the terminal commands executed by the provided plugin work as expected.

**Pull request merged**